source/app.js

```diff
@@ -133,7 +133,8 @@ class Application {
                     'pkl', 'joblib',
                     'pbtxt', 'prototxt',
                     'cfg',
-                    'xml' ] }
+                    'xml',
+                    'ms' ] }
```

Comment:
`.ms` is ambiguous. Please work with the team to change this to a more descriptive extension like `.mslite`.

Reply:
Since the extension of our many models is .ms, changing it will have a great impact. Instead, in the line 38 of mslite.js, I've supplement a format matching to determine whether a ’.ms' file is a MindSpore Lite model.